weblate_web/templates/user.html

```diff
@@ -64,38 +76,41 @@ <h1>{% trans "Your donations" %}</h1>
                         {% endif %}
                         <form class="form-inline" method="post" action="{% url "donate-pay" pk=donation.pk %}">
                         {% csrf_token %}
-                        <input type="submit" class="button small inline" value="{% trans "Renew now" %}">
+                        <input type="submit" class="button med inline" value="{% trans "Renew now" %}">
                         </form>
                         {% if donation.active and donation.payment_obj.recurring %}
                         <form class="form-inline" method="post" action="{% url "donate-disable" pk=donation.pk %}">
                         {% csrf_token %}
-                        <input type="submit" class="button small inline" value="{% trans "Turn off renewal" %}">
+                        <input type="submit" class="button med inline" value="{% trans "Turn off renewal" %}">
                         </form>
                         {% endif %}
                     </div>
                     <div class="clear"></div>
                 </div>
-                <div class="form-line">
-                    <div class="line-left">{% trans "Payments" %}</div>
-                    <div class="line-right">
+                <div class="form-line style2">
+                    <div class="line-left style4">{% trans "Payments" %}</div>
+                    <div class="line-right style3">
                     {% for payment in donation.list_payments %}
                     {% include "snippets/payment.html" %}
```

Comment:
The payments for donation are wrongly rendered:

![image](https://user-images.githubusercontent.com/212189/106595973-aed84780-6554-11eb-99e5-58db2d279d45.png)

Reply:
Do you mean design is incorrect or functionality ?

Reply:
I see there is not enough room which is why it's showing like this. I'll try to adjust it

Reply:
I have fixed that.